src/backend/marsha/core/tests/test_lti_user_association.py

```diff
@@ -0,0 +1,120 @@
+"""Test the user_association module from core.lti."""
+import uuid
```

Comment:
What do you think about using keyword arguments for all the tests when you call a function from `marsha.core.lti.user_association` ? It will ease the read and remember what argument is what.